website/client/src/components/messages/messageCard.vue

```diff
@@ -123,49 +238,173 @@
       min-height: 0rem;
     }
   }
+}
 
-  hr {
-    margin-bottom: 0.5rem;
-    margin-top: 0.5rem;
+.card-menu {
+  position: absolute;
+  top: 0;
+  right: 0;
+
+  &:not(.show) {
+    display: none;
+  }
+}
+
+.card-body:hover {
+  .card-menu {
+    display: block;
   }
+}
+
+hr {
+  margin-bottom: 0.5rem;
+  margin-top: 0.5rem;
+}
+
+.reported {
+  margin-top: 18px;
+  color: $red-50;
+}
 
-  .reported {
-    margin-top: 18px;
-    color: $red-50;
+.selectListItem:not(:hover) .svg-icon.icon-16.color {
+  color: #{$gray-100}
+}
+
+.custom-hover--red {
+  --hover-color: #{$maroon-50};
+  --hover-background: #{rgba($red-500, 0.25)};
+}
+
+.user-sent-message {
+  border: 1px solid $purple-400;
+}
+
+.system-message {
+  border: 1px solid $purple-400;
+}
+
+.user-received-message {
+  border: 1px solid $gray-500;
+}
+
+.card-menu {
+  // icon-color is the menu icon itsem
```

Comment:
...itsem?

Reply:
well... you know right... itsem ... common... its known.. 😬  - fixing hahha